apps/angular/src/app/features/anime/anime-table/anime-table.component.html

```diff
@@ -0,0 +1,60 @@
+<table
+  class="table"
+  *ngIf="animeList$ | async as animeList; else loading"
+  mat-table
+  [dataSource]="animeList"
+  [trackBy]="trackAnimeById"
+  class="mat-elevation-z8"
+>
+  <caption class="table__title">
+    Anime collection
+  </caption>
+  <ng-container matColumnDef="Image">
+    <th class="table__head-title" mat-header-cell *matHeaderCellDef>Image</th>
+    <td mat-cell *matCellDef="let anime">
+      <img class="table__row-image" src="{{ anime.image }}" alt="" />
```

Comment:
Usually we prefer using a bit different syntax:
```suggestion
      <img class="table__row-image" [src]="anime.image" alt="" />
```